specs/001-kiro-specs-knowledge/quickstart.md

```diff
@@ -0,0 +1,187 @@
+# Knowledge Graph System Quickstart
+
+## Prerequisites
+- Python 3.12+
+- pixi environment setup
+- NWB file for testing
+- NWB-LinkML schema available
+
+## Quick Start Steps
+
+### 1. Environment Setup
+```bash
+# Activate pixi environment
+pixi shell
+
+# Verify dependencies
+python -c "import rdflib, linkml, pynwb; print('Dependencies OK')"
+```
+
+### 2. Create Knowledge Graph from NWB File
+```bash
+# Using the MCP tool endpoint
+curl -X POST http://localhost:8000/knowledge-graphs \
+  -H "Content-Type: application/json" \
+  -d '{
+    "nwb_file_path": "/path/to/sample.nwb",
+    "schema_version": "2.8.0"
+  }'
+```
+
+Expected response:
+```json
+{
+  "graph_id": "kg-12345",
+  "status": "building",
+  "nwb_file_path": "/path/to/sample.nwb",
+  "schema_version": "2.8.0",
+  "created_at": "2025-09-25T19:30:00Z"
+}
+```
+
+### 3. Check Graph Status
+```bash
+curl http://localhost:8000/knowledge-graphs/kg-12345
```

Comment:
I really would love to do this sort of stuff with python. Jumping between shell and python will slow the human down. As long as the AI knows to single quote python blocks it can usually do well with it. Shell stuff can be very concise but they don't naturally migrate into a pytest suite.